Lib/idlelib/idle_test/htest.py

```diff
@@ -335,7 +335,7 @@ def _wrapper(parent):  # htest #
 _widget_redirector_spec = {
     'file': 'redirector',
     'kwds': {},
-    'msg': "Every text insert should be printed to the console."
+    'msg': "Every text insert should be printed to the console. "
```

Comment:
I realize the period after `console` was already on this line, but does it make any sense? I think it should be deleted, and the new end of this line should be `console "`.